web/themes/custom/novel/templates/components/list-teaser.html.twig

```diff
@@ -0,0 +1,49 @@
+{%
+  set classes = [
+  'content-list-item',
+  'arrow__hover--right-small',
+]
+%}
+
+<a {{ attributes.addClass(classes).setAttribute('href', url) }}>
+  <div class="content-list-item__image-container">
+    {{ image }}
+  </div>
+  <div class="content-list-item__content">
+    {{ categories }}
+    {% if date %}
+      <div class="content-list-item__date">
+        {{ date }}
+      </div>
+    {% endif %}
+
+    <h3 class="content-list-item__title">{{ title }}</h3>
```

Comment:
Please make this a `<h2>`.

With regard to accessibility and with the latest changes to the design system this should be right heading level.